README.md

```diff
@@ -11,64 +9,76 @@ Contents
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/1.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/2.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/3.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/4.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/5.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/6.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/7.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/8.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+<img border="0" width="1" height="1" src="http://traverse.example.com/v1/YOUR-CLIENT-ID-HERE/9.gif?emailMd5Lower=f3ada405ce890b6f8204094deb12d8a8&emailSha1Lower=823776525776c8f23a87176c59d25759da7a52c4&emailSha256Lower=0c7e6a405862e402eb76a70f8a26fc732d07c32931e9fae9ab1582911d2e8a3b&domain=bar.com"/>
+```
 
+__Note__: We still support supplying one hash per pixel and repeating indexes for each hash type. This is not encouraged anymore. Providing all hashes available to each pixel will provide the most revenue. Example:
```

Comment:
I think it's important to indicate that this still works, as all of our clients have this way